apps/api/src/app/events/usecases/send-message/send-message.usecase.ts

```diff
@@ -174,11 +175,15 @@ export class SendMessage {
       environmentId: job._environmentId,
     });
 
+    const subscriber = await this.subscriberRepository.findById(job._subscriberId);
```

Comment:
In this PR #2489 we added user's subscriberId to the job entity. I will add a not note on this PR to get the subscriber from the cache first.

Reply:
Awesome so we can fix it there later 🎉

Reply:
Yeah i left a note there so we wont forget 😅